public/js/components/stories/Frames.js

```diff
@@ -28,8 +34,8 @@ function renderContainer(Component, frames) {
       margin: "auto",
       paddingTop: "100px"
     }},
-    dom.div({ style: { border: "1px solid #ccc", padding: "20px" }},
-            createElement(Provider, { store },
-                          createElement(Component)))
+    createElement(Provider, { store },
+      dom.div({ className: "accordion" },
```

Comment:
Why are you wrapping this with an accordion class? If these are to be as docs that's sort of confusing.